lib/seattleflu/id3c/cli/command/__init__.py

```diff
@@ -7,6 +7,113 @@
 By in turn loading our own individual commands here, we allow each command
 module to register itself via Click's decorators.
 """
-from . import (
-    reportable_conditions
-)
+import pandas as pd
+from typing import List
+
+# Load all ETL subcommands.
+__all__ = [
+    "clinical",
+    "longitudinal",
+    "reportable_conditions"
+]
+
```

Comment:
Here, I went with the "old style" of `__init__` subcommand loading because
1. I don't fully understand how the "new style" works, and 
2. I had a boatload of other helper functions to import.

Thoughts? Suggestions?

Reply:
I'm not sure what you mean by new vs. old style…? I think the code looks fine, though.